Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -78,12 +80,35 @@ namespace ROS2
 
     void RobotImporterWidget::OpenUrdf()
     {
-        m_urdfPath = AZStd::string(m_fileSelectPage->getFileName().toUtf8().constData());
+        QString report;
         if (!m_urdfPath.empty())
         {
-            AZ_Printf("Wizard", "Loading URDF file : %s", m_urdfPath.c_str());
-            m_parsedUrdf = UrdfParser::ParseFromFile(m_urdfPath);
-            QString report;
+            if (m_urdfPath.ends_with("xacro"))
```

Comment:
suggestion: For now this is fine but if in future there are more urdf formats we might want to consider refactoring this so it is easy to add new cases without modifying an if/else or switch block (if there's just there's two it's definitely not worth it but just something that could help maintainability in future potentially